docs/source/quantization.rst

```diff
@@ -38,20 +38,177 @@ that perform all or part of the computation in lower precision. Higher-level
 APIs are provided that incorporate typical workflows of converting FP32 model
 to lower precision with minimal accuracy loss.
 
-Quantization requires users to be aware of three concepts:
+Quantization Stack
+--------------------------------------
+Quantization is the process to convert a floating point model to a quantized model. So at high level the quantization stack can be split into two parts: 1). The building blocks or abstractions for a quantized model 2). The building blocks or abstractions for the quantization flow that converts a floating point model to a quantized model
 
-#. Quantization Config (Qconfig): Specifies how weights and activations are to be quantized. Qconfig is needed to create a quantized model.
-#. Backend: Refers to kernels that support quantization, usually with different numerics.
-#. Quantization engine (torch.backends.quantization.engine): When a quantized model is executed, the qengine specifies which backend is to be used for execution. It is important to ensure that the qengine is consistent with the Qconfig.
+Quantized Model
+^^^^^^^^^^^^^^^^^^^^^^^
+Quantized Tensor
+~~~~~~~~~~~~~~~~~
+We need a representation for quantized activations and weights in a quantized model. A Quantized Tensor is a type of a PyTorch Tensor. It is quantized from a Tensor with higher precisions like float. And it stores quantization parameters like scale and zero_point.
 
-Quantization API Summary
```

Comment:
IMO the API summary should be as close to the top as possible, and people shoudl scroll down if they need more details